src/font/mod.rs

```diff
@@ -1,109 +1,54 @@
 // SPDX-License-Identifier: MIT OR Apache-2.0
 pub(crate) mod fallback;
 
-use alloc::boxed::Box;
+use core::fmt;
+
 use alloc::sync::Arc;
 
 pub use self::system::*;
 mod system;
 
-pub use font_inner::Font;
+mod owned_face {
+    impl_self_ref!(OwnedFace, rustybuzz::Face<'static>, rustybuzz::Face<'this>);
+}
+use owned_face::*;
```

Comment:
wrapping this up in a module is necessary to prevent accessing the fields incorrectly so it should probably be part of the macro